Trio/Sources/Modules/Home/View/HomeRootView.swift

```diff
@@ -467,31 +496,38 @@ extension Home {
                         .font(.callout)
                 } else {
                     HStack {
-                        if state.pumpSuspended {
+                        if state.apsManager?.isScheduledBasal == nil {
+                            /// The pump is not currently available (e.g., no pod) so
+                            /// display no insulin delivery info rather than "Pump suspended"
+                        } else if state.apsManager.isSuspended {
```

Comment:
If the pump is suspended, let's display the last rate that was active and style it striked through. 

Depending on localization and text enlargement / zoom factor, this label here will break the display. 

I realize this isn't a thing brought in with this PR, but please make this change, i.e., when pump is suspended, display the TBR / SBR rate as

~~💧 1.0 U/hr~~

Reply:
I agree, that just displaying a rate without _any_ change in the UI might implicate that the rate is still running (it creates inconsistency in the UI of the header), so would be wrong.

However, when the pump is suspended, it already displays a big fat "Insulin suspended"to the left to the "glucose bobble". IMO it's unnecessary to display the suspension state twice.  Hence, it will say that it's suspended to the left for the "pump info area", and when it is in this state, it displays the last "correct" rate (you introduced this fix with this PR), so tells the user what was happening just before the suspension, yet strikes this value, as the suspension withholds any insulin from being dosed.